typescript/sdk/src/token/Token.ts

```diff
@@ -341,6 +342,49 @@ export class Token implements IToken {
     return new TokenAmount(balance, this);
   }
 
+  /**
+   * Gets the amount of tokens bridged, if possible.
+   * @param multiProvider A multiProvider that contains the chain for this token.
+   * @returns The amount of tokens bridged, or undefined if not supported.
+   */
+  async getBridgedSupply(
```

Comment:
This is fine for now but IMO this _feels_ like something that we'd add to `IHypTokenAdapter` and then implement in the adapters, rather than a big function here branching based on standard. That would also remove the need for casting.

Reply:
agreed - done!